lib/autoload/course/assessment/programming_test_case_report.rb

```diff
@@ -225,7 +225,11 @@ def messages
     # @return [String]
     def get_meta_attribute(attribute_name)
       meta = @test_case.search('meta')
-      meta.present? && meta[0][attribute_name] ? meta[0][attribute_name] : ''
+      if meta.present? && meta[0][attribute_name]
```

Comment:
Making the comments clearer is fine too - just raising a point that if I misunderstood `get_meta_attribute` as get attributes from meta tag rather than get meta-attributes, I'm sure there will be more like me who might take a while to figure out the function 😅 

Also, are the 'specifications' for the test unit outputs written anywhere?